src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -354,6 +354,8 @@ def load_arguments(self, _):
         c.argument('enable_managed_identity', action='store_true')
         c.argument('assign_identity', type=str,
                    validator=validate_assign_identity)
+        c.argument('assign_kubelet_identity', type=str,
+                   validator=validate_assign_kubelet_identity)
```

Comment:
`str` is the default value of `assign_kubelet_identity`, so there is no need to add it repeatedly
```suggestion
        c.argument('assign_kubelet_identity',
                   validator=validate_assign_kubelet_identity)
```